tools/site_and_regional/subset_data

```diff
@@ -9,17 +9,14 @@ For full instructions on how to run the code and different options,
 please check python/ctsm/subset_data.py file.
 This script extracts domain files, surface dataset, and DATM files
 at either a single point or a region using the global dataset.
-
```

Comment:
@adrifoster could you restore this? Not a big deal...

Reply:
I thought I added that back! Not sure why it keeps disappearing...